src/kafka-util/src/addr.rs

```diff
@@ -48,6 +53,44 @@ impl fmt::Display for KafkaAddrs {
     }
 }
 
+impl From<&(String, u16)> for proto_kafka_addrs::ProtoKafkaAddr {
```

Comment:
why can't be put the proto and impls in `mz-dataflow-types`? `kafka-util` seems like a weird place to do proto stuff

Reply:
No this is right! We're keeping the Protobuf definitions alongside the Rust definitions, whatever crate that may be!

Reply:
Two things:
1. [Our convention](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/20220511_command_and_response_binary_encoding.md) is to implement the Protobuf conversion logic always together with the original types in their original crates, instead of having a single Protobuf conversion crate or something similar.
2. My understanding is that since `From`/`TryFrom` are in the stdlib, I could only implement these traits for `KafkaAddrs` in the same file where `KafkaAddrs` is defined.